internal/host/validator.go

```diff
@@ -977,8 +1004,9 @@ func (v *validator) hasSufficientPacketLossRequirementForRole(c *validationConte
 	case ValidationSuccess:
 		return status, "Packet loss requirement has been satisfied."
 	case ValidationFailure:
-
-		return status, fmt.Sprintf("Packet loss percentage requirement of %s %.2f%% not met for connectivity between %s and%s.", comparisonBuilder(*c.clusterHostRequirements.Total.PacketLossPercentage), *c.clusterHostRequirements.Total.PacketLossPercentage, c.host.ID, strings.Join(hostPacketLoss, ","))
+		packetLossAdvisoryMessage := fmt.Sprintf("A total packet loss above the tolerated threshold of %.2f%% was encountered when performing connectivity validation between host %s and%s", *c.clusterHostRequirements.Total.PacketLossPercentage, c.host.ID, strings.Join(hostPacketLoss, ","))
```

Comment:
and%s

Reply:
oh actually this seems to be on purpose, nevermind

Reply:
NIT, missing space in `between host %s and%s` ?